tests/queue_test.py

```diff
@@ -97,6 +97,11 @@ async def queue_put():
 
 
 class ManagerQueueTest(BaseTest):
+    @unittest.skipIf(
+        "multiprocess.util" in str(util),
+        "concurrent.futures is not yet supported by uqfoundation "
+        "(https://github.com/uqfoundation/pathos/issues/90)"
+    )
```

Comment:
sub-optimal to skip this test... especially for people that are using `concurrent.futures.ProcessPoolExecutor` in combination with `aioprocessing`, in an environment where `multiprocess` happens to be installed. As the author stated in https://github.com/uqfoundation/pathos/issues/90, it should be an easy PR. I've tried figuring it out multiple times as I keep running into multiprocessing problems in async contexts, but repeatedly stranded.

Here's the traceback for this test case, caused by ProcessPoolExecutor internally using `multiprocessing` instead of `multiprocess`:
```pytb
test_executor (tests.queue_test.ManagerQueueTest) ... Process SpawnProcess-1:
Traceback (most recent call last):
  File "~/.pyenv/versions/3.8.6/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "~/.pyenv/versions/3.8.6/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "~/.pyenv/versions/3.8.6/lib/python3.8/concurrent/futures/process.py", line 233, in _process_worker
    call_item = call_queue.get(block=True)
  File "~/.pyenv/versions/3.8.6/lib/python3.8/multiprocessing/queues.py", line 116, in get
    return _ForkingPickler.loads(res)
  File "~/.pyenv/versions/pi/lib/python3.8/site-packages/multiprocess/managers.py", line 959, in RebuildProxy
    return func(token, serializer, incref=incref, **kwds)
  File "~/.pyenv/versions/pi/lib/python3.8/site-packages/multiprocess/managers.py", line 809, in __init__
    self._incref()
  File "~/.pyenv/versions/pi/lib/python3.8/site-packages/multiprocess/managers.py", line 863, in _incref
    conn = self._Client(self._token.address, authkey=self._authkey)
  File "~/.pyenv/versions/pi/lib/python3.8/site-packages/multiprocess/connection.py", line 511, in Client
    answer_challenge(c, authkey)
  File "~/.pyenv/versions/pi/lib/python3.8/site-packages/multiprocess/connection.py", line 762, in answer_challenge
    raise AuthenticationError('digest sent was rejected')
multiprocess.context.AuthenticationError: digest sent was rejected
```

Reply:
note that even if uqfoundation would add compatibility, and expose `pathos.futures.ProcessPoolExecutor` or so, this error would still persist for people trying to combine the `concurrent.futures.ProcessPoolExecutor` with `aioprocessing[dill]`.

Reply:
I think there should be a note in the readme about this incompatibility when using dill, as well as documenting the workaround that forces usage of `multiprocessing`. People upgrading to this version could suddenly have their code break, so it should be very clear how to fix it.

Reply:
Sure thing, added. Assuming semver and this being a potentially breaking change ^ I went ahead and bumped to 2.0.0 🤙